tests/integration/orm/mixed/2-many-to-one/accessor-test.js

```diff
@@ -14,7 +14,6 @@ module('Integration | ORM | Mixed | Many To One | accessor', function(hooks) {
     test(`the references of a ${state} are correct`, function(assert) {
       let [ post, user ] = this.helper[state]();
 
-      assert.deepEqual(post.user, user);
```

Comment:
A regular `equal` call had issues here too. Do you have any suggestions?

Reply:
Hm try `assert.ok(post.user.equals(user))`

Reply:
Ah I see. I might do something like this:
```
if (post.user) {
  assert.ok(post.user.equals(user));
} else {
  assert.equal(post.user, null);
  assert.equal(user, null);
}
```